index.test.ts

```diff
@@ -143,6 +144,31 @@ beforeAll(() => {
         expire: redis.expire.bind(redis),
         get: (key: string, defaultValue: unknown) => redis.get(key)
     }
+
+    mockStorage = new Map()
+    storage = {
+        // Based of https://github.com/PostHog/posthog/blob/master/plugin-server/src/worker/vm/extensions/storage.ts
+        get: async function (key: string, defaultValue: unknown): Promise<unknown> {
```

Comment:
Is TS not complaining that this async function has no await?

Reply:
that's the tricky thing here - that we could easily have a mistake in the code, where we don't await (though TS would complain), is there a way to pretend it's an async function or should I just drop async here?